quiz.py

```diff
@@ -1,6 +1,65 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Sebastian Sanchez
+Date: 2024/03/04
+Description: Multiple choice quiz with final score
+"""
+
+print("-------------------------------")
+print("Multiple-Choice Quiz Game")
+
+#Initialize score
+score = 0
+
+#First question
+print("\n1. What is the power house of the cell?")
```

Comment:
You clearly know how to use the `\n` special character. Can you do this output with a single `print()` statement? Furthermore, can you use the escape character `\` to span the statement across multiple lines to make it easily readable? For example, this is difficult to read:

```python
print("What is the answer?\n(a) Choice\n(b) Choice\n(C) Choice")
```
But this is the exact same output, nicely formatted:

```python
print("What is the answer? \
      \n(a) Choice \
      \n(b) Choice \
      \n(C) Choice \
      ")
```